core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb_test_export.go

```diff
@@ -87,12 +97,14 @@ func (env *testVDBEnv) cleanup() {
 }
 
 // CleanupDB deletes all the databases other than fabric internal database
-func CleanupDB(t testing.TB, dbProvider statedb.VersionedDBProvider) {
-	cleanupDB(t, dbProvider.(*VersionedDBProvider).couchInstance)
+func CleanupDB(t testing.TB, config *ledger.CouchDBConfig) {
+	couchInstance, err := createCouchInstance(config, &disabled.Provider{})
+	require.NoError(t, err)
+	cleanupDB(t, couchInstance)
 }
 
-func cleanupDB(t testing.TB, couchInstance *couchdb.CouchInstance) {
-	dbNames, err := couchInstance.RetrieveApplicationDBNames()
+func cleanupDB(t testing.TB, couchInstance *couchInstance) {
```

Comment:
Why do you need both `CleanupDB` and `cleanupDB` ?

Reply:
Yes, Gari. I had the same question. As the code existed earlier, I left it as is. I will fix it.